api/src/pcapi/routes/backoffice/venues/blueprint.py

```diff
@@ -491,6 +495,30 @@ def get_invoices(venue_id: int) -> utils.BackofficeResponse:
         "venue/get/invoices.html",
         reimbursement_point=current_link.reimbursementPoint if current_link else None,
         invoices=invoices,
+        venue_id=venue_id,
+    )
+
+
+@venue_blueprint.route("/<int:venue_id>/reimbursement-details", methods=["POST"])
+def download_reimbursement_details(venue_id: int) -> utils.BackofficeResponse:
+    form = empty_forms.BatchForm()
+    if not form.validate():
+        flash(utils.build_form_error_msg(form), "warning")
+        return redirect(request.referrer or url_for(".list_venues"), code=303)
+
+    invoices = finance_models.Invoice.query.filter(finance_models.Invoice.id.in_(form.object_ids_list)).all()
+
+    reimbursement_details = [
+        reimbursement_csv_serialize.ReimbursementDetails(details)
+        for details in finance_repository.find_all_invoices_finance_details([invoice.id for invoice in invoices])
+    ]
+    export_data = reimbursement_csv_serialize.generate_reimbursement_details_csv(reimbursement_details)
+    export_date = datetime.utcnow().strftime("%Y-%m-%dT%H:%M")
```

Comment:
Je suis partisan d'éviter les caractères interdits sur les systèmes de fichiers Microsoft, donc d'avoir des tirets partout : https://learn.microsoft.com/fr-fr/windows/win32/fileio/naming-a-file
```suggestion
    export_date = datetime.utcnow().strftime("%Y-%m-%d-%H-%M")
```